### ISSUE TITLE
Vader segvs on master (PPC)

### ISSUE THREAD

**Issue by jsquyres** [open-mpi/ompi#5394]
On master, we're seeing vader errors since the vader BTL RDMA changes went in recently.  Here's an example of MTT fails from over the past weekend https://mtt.open-mpi.org/index.php?do_redir=2647:

```
seed value: 484819717
[c656f6n04:137166] *** An error occurred in MPI_Win_create
[c656f6n04:137166] *** reported by process [1045561345,1]
[c656f6n04:137166] *** on communicator MPI_COMM_WORLD
[c656f6n04:137166] *** MPI_ERR_WIN: invalid window
[c656f6n04:137166] *** MPI_ERRORS_ARE_FATAL (processes in this communicator will now abort,
[c656f6n04:137166] ***    and potentially your MPI job)[c656f6n04:137159] 1 more process has sent
help message help-mpi-errors.txt / mpi_errors_are_fatal
[c656f6n04:137159] Set MCA parameter "orte_base_help_aggregate" to 0 to see all help / error
messages
```

@rhc54 talked to @hjelmn; Nathan is supposedly fixing the issue now.

**Comment by jsquyres** [comment]
@PeterGottesman @hjelmn Is this now fixed?

**Comment by PeterGottesman** [comment]
I am unable to check this since it is on PPC64

**Comment by jsquyres** [comment]
According to @hjelmn this was actually an issue in OSC RDMA, and is has been fixed (before the v4.0.x branch was created).
